contrib/local/configs/keys.json

```diff
@@ -3,7 +3,7 @@
     {
       "name": "test1",
       "type": "local",
-      "address": "persistence1ht0tun4u5uj4f4z83p9tncjerwu27ycsm52txm",
+      "address": "persistence152nvc6f096v6n6tr5lg50xq22ak0chsr0ru7xc",
       "pubkey": {
         "@type": "/cosmos.crypto.secp256k1.PubKey",
         "key": "AoC0SFBrAyiHrtJ+uGFuxP9CyTK8zSppylBhB+KBONNq"
```

Comment:
pubkeys should also be updated for all

Reply:
@puneet2019  The pubkey should remain same post cointype change as well right?
I tried with the latest `persistenceCore` binary, for the same `mnemonic` the public key remains same but the address changes to what @Anmol1696 has updated it to. Is that a bug in the persistenceCore binary or indeed the intended behavior?

Reply:
I checked and the pubKeys remain the same

Reply:
thanks! looks like a bug when keys were taken from gaia.. the pubkeys were not updated then